cylc/flow/cfgspec/workflow.py

```diff
@@ -1406,7 +1429,42 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     determine how an event handler responds to task failure
                     events.
                 ''')
+            with Conf('skip', desc='''
+                Task configuration for task :ref:`task-run-modes.skip`.
 
+                For a full description of skip run mode see
+                :ref:`task-run-modes.skip`.
+
+                .. versionadded:: 8.4.0
+            '''):
+                Conf(
+                    'outputs',
+                    VDR.V_STRING_LIST,
+                    desc='''
+                        Outputs to be emitted by a task in skip mode.
+
+                        * By default, all required outputs will be generated
+                          plus succeeded if success is optional.
+                        * If outputs is specified and does not include either
```

Comment:
Worth clarifying "outputs" in this context for user-guide purposes, e.g:

```suggestion
                        * If skip-mode outputs is specified and does not include either
```